### DISCUSSION TITLE
posts/project-init-on-cloudflare-pages/

### DISCUSSION THREAD

**Discussion by giscus** [yunnysunny/yunnysunny.github.io#29] [Show and tell]
# posts/project-init-on-cloudflare-pages/

在之前的文章中，谈到了笔者的博客被攻击，在尝试了各种国内 CDN 解决方案后都无解，最后靠 cloudflare 提供的免费 page 服务完美解决。从这篇文章起，笔者将开始深入探究 page 的使用方法，让更多的人能够用上这种惠民的服务。作为开篇之作，本文重点关注 cloudflare page 的项目初始化和部署相关的内容。

https://blog.whyun.com/posts/project-init-on-cloudflare-pages/

**Comment by hticy3214** [comment]
博主你好，感谢分享使用方法，由于CF在国内并不普及，且阅读英文文档有很大限制，文中提到的一些特性，官方文档内很难找到，还有国内使用wrangler的问题。对于workers和pages，官方支持很多框架，这些框架看文档很好用很轻便，但实际使用起来，一是像文中提到的能做什么不能做什么，很难从文档内找到。二是在使用起来，很多细节描述的并不清楚，按官方文档的代码都存在运行失败的情况。

**Comment by ghost** [comment]
请问下这个和vercel比的话哪个好点

**Reply by yunnysunny** [reply]
vercel有免费流量，但是流量有上限，如果你的网站被ddos的话，用cloudflare pages会好些；还有就是vercel的默认子域名在国内貌似无法直接访问，绑定自定义域名后可以访问。
其他方面的考量，我没有调研过，比如说第三方前端框架的兼容性方面。

**Reply by ghost** [reply]
谢谢大佬解惑😃
